src/javascript/SelectorTypes/Category/category.adapter.js

```diff
@@ -1,4 +1,11 @@
 export const adaptToCategoryTree = ({nodes, parent, selectedValues, locale}) => {
+
+    // This is only to raise the issue, you will most likely want to fix it somewhere else and do something nicer as well.
+    // localeCompare in JS expect a locale like en-US NOT en_US which Jahia uses.
+    if (locale.indexOf("_") !== -1) {
+        locale = locale.replace("_", "-");
+    }
```

Comment:
Maybe use replaceAll instead:

```
if (locale.indexOf('_') !== -1) {
    locale = locale.replaceAll('_', '-');
}
```

Reply:
Also need to use single vs double quotes for strings (for lint purposes)

Reply:
Sure for the single quote, but I don't see the point of a replaceAll here?
The replace function is faster and the string can't have 2 underscores so I don't see the point.

Reply:
I believe it's technically possible for locales in general to have multiple underscores/hyphens, to support dialects for example so it's the safer route but if we think that's not something we currently support in Jahia then I guess it's also ok